addons/html_editor/static/src/editor/utils/selection.js

```diff
@@ -390,6 +391,10 @@ export function getTraversedNodes(editable, range = getDeepRange(editable)) {
     return [...traversedNodes];
 }
 
+export function getTraversedBlocks(editable, range = getDeepRange(editable)) {
```

Comment:
maybe rename this to `getSelectedBlocks`, it seems more intuitive to me

Reply:
@ged-odoo 
Yes... but:
We have already `getTraversedNodes` and `getSelectedNodes`,  being the 2nd a subset of the 1st: " Returns an array containing all the nodes **fully** contained in the selection".
As this new function returns something that relates more to what `getTraversedNodes` does (partially selected blocks are returned), I decided for such name (`getTraversedBlocks`).

Would you still like me to change to `getSelectedBlocks`?